app/services/update_auction.rb

```diff
@@ -10,11 +10,31 @@ def perform
   end
 
   def should_create_cap_proposal?
-    attributes[:result] == 'accepted' && auction.cap_proposal_url.blank?
+    auction_accepted_and_cap_proposal_is_blank? &&
+      winning_bidder_is_eligible_to_be_paid?
   end
 
   private
 
+  def auction_accepted_and_cap_proposal_is_blank?
+    attributes[:result] == 'accepted' && auction.cap_proposal_url.blank?
+  end
+
+  def winning_bidder_is_eligible_to_be_paid?
+    auction_presenter = AuctionPresenter.new(auction)
+    if auction_presenter.small_business?
+      winning_bidder = auction_presenter.winning_bid.bidder
+
+      reckoner = SamAccountReckoner.new(winning_bidder)
+      reckoner.set!
+      winning_bidder.reload
+
+      auction_presenter.user_is_eligible_to_bid?(winning_bidder)
```

Comment:
I think it should! The UI notification is part of this story https://github.com/18F/micropurchase/issues/574